include/polygeist/Passes/Passes.td

```diff
@@ -4,6 +4,16 @@
 include "mlir/Pass/PassBase.td"
 include "mlir/Rewrite/PassUtil.td"
 
+def AffineOpt : Pass<"affine-opt"> {
```

Comment:
maybe call it pluto opt?

Reply:
also okay with the name tho

Reply:
Maybe polyhedral-opt because in a later wip commit we will introduce a different scheduler as an option to this pass

Reply:
@wsmoses Done